torchtext/experimental/models/roberta_transform.py

```diff
@@ -0,0 +1,28 @@
+import os
+import torch.nn as nn
+from torchtext.experimental.transforms import sentencepiece_tokenizer
+from torchtext.experimental.vocab import load_vocab_from_file
+from typing import List
+
+
+class RobertaTransform(nn.Module):
+    """Roberta encode transform."""
+
+    def __init__(self, tokenizer, vocab):
+        super(RobertaTransform, self).__init__()
+        self.tokenizer = tokenizer
+        self.vocab = vocab
+
+    @classmethod
+    def from_pretrained(cls, directory='./',
```

Comment:
same comment here about removing `directory` and having full paths for `tokenizer` and `vocab`